### PR TITLE
Project 11

### PR DESCRIPTION
**Pull request by volodymyrtaras**

### CODE REVIEW

**Comment by volodymyrtaras** [comment]
@Anderwerty привіт) можеш глянути моє дз, Саша сказав, що до кінця лекції заапрувиш, то зарахує)

File: src/main/java/eleven/Util.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+public class Util {
+    private static final Logger LOGGER = Logger.getLogger(Util.class);
+    private static final Gson GSON = new GsonBuilder().setPrettyPrinting().setDateFormat("dd/MM/yyyy").create();
+
+    public static void writeToFile(List list, String path) {
+        try (Writer writer = new FileWriter(path)) {
+            GSON.toJson(list, writer);
+        } catch (IOException e) {
+            LOGGER.info(e.getMessage());
```
Comment (Anderwerty) [inline src/main/java/eleven/Util.java]: Maybe not info but error or at least warn

File: src/main/java/eleven/Util.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        }
+    }
+
+    public static List<Fruit> readFromFile(String path) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(path))) {
+            Type type = new TypeToken<ArrayList<Fruit>>() {
+            }.getType();
+            return GSON.fromJson(bufferedReader, type);
+        } catch (IOException e) {
+            LOGGER.info(e.getMessage());
```
Comment (Anderwerty) [inline src/main/java/eleven/Util.java]: The same

File: src/test/java/eleven/StoreTest.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class StoreTest {
```
Comment (Anderwerty) [inline src/test/java/eleven/StoreTest.java]: Did you add it on the web version?

File: .gitignore [hunk 4]
```diff
@@ -1,8 +1,7 @@
 .idea
-out
 target
+log
 *.xml
```
Comment (Anderwerty) [inline .gitignore]: It is not right. In this case, you will ignore pom.xml

File: .gitignore [hunk 5]
```diff
@@ -1,8 +1,7 @@
...
 *.iml
 *.class
```
Comment (Anderwerty) [inline .gitignore]: All files with extension *.class will be in package target. So this ignore is redundant.

File: .gitignore [hunk 6]
```diff
@@ -1,8 +1,7 @@
...
-*.txt
 *.attach
```
Comment (Anderwerty) [inline .gitignore]: Please explain with you add it into git ignore? And please add a blank line at the end of the file.

Comment (volodymyrtaras) [reply .gitignore]: @Anderwerty I have instructions in txt-files

File: .gitignore (hunk 4 above)
Comment (volodymyrtaras) [reply .gitignore]: @Anderwerty I added this before because of another TA (Taniann) think that these files are redundant

**Pull request merged**